coop_default_pricetag/models/product_template.py

```diff
@@ -161,14 +159,14 @@ def _compute_price_volume(self):
             else:
                 data.price_volume = ""
 
-    @api.depends("list_price", "weight_net")
+    @api.depends("list_price", "weight")
     @api.multi
     def _compute_price_weight_net(self):
         """Return the price by kg"""
         for data in self:
-            if data.list_price and data.weight_net:
+            if data.list_price and data.weight:
                 data.price_weight_net = "%.2f" % round(
```

Comment:
Since we're changing the field type from Char to Monetary, let's not convert to string and let Odoo handle rounding.